integrations/linear/.port/spec.yaml

```diff
@@ -0,0 +1,22 @@
+version: v0.1.0
```

Comment:
Remove this line
Make sure this file is aligned to the latest version

Reply:
Removed and checked with more recent integration spec.yaml files